client/opt/option.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package pd
+package opt
```

Comment:
Will it have a compatibility issue?

Reply:
Yes, when upgrading the downstream components, manual modification of the relevant code is required.